cmd/virt-chroot/main.go

```diff
@@ -9,6 +11,12 @@ import (
 	"strings"
 	"syscall"
 
+	runc_fs2 "github.com/opencontainers/runc/libcontainer/cgroups/fs2"
```

Comment:
@iholder-redhat Hi! a small remark here - maybe for consistency reason we should not use snake case in the package name aliasing, because we may end up with a mix of camel case and snake case.